tests/unit/test_network_connections.py

```diff
@@ -2998,6 +2998,106 @@ def test_valid_persistent_state(self):
         input_connection.update({"state": "absent"})
         self.assertValidationError(validator, input_connection)
 
+    def test_set_deprecated_master(self):
+        """
+        When passing the deprecated "master" it is updated to "controller".
+        """
+        input_connections = [
+            {
+                "name": "prod2",
+                "state": "up",
+                "type": "bridge",
+                "interface_name": "bridge2",
+                "ip": {"dhcp4": False, "auto6": False},
+            },
+            {
+                "name": "prod2-slave1",
+                "state": "up",
+                "type": "ethernet",
+                "interface_name": "eth1",
+                "master": "prod2",
+            },
+        ]
+        expected_connections = [
```

Comment:
This big list of expected connections. Would it be possible to make it as concise as the other tests around deprecation like https://github.com/linux-system-roles/network/blob/main/tests/unit/test_network_connections.py#L2951 ?

In this case it would be nice to only:
1. Assert that controller: "prod2" is in the output of the validation
2. Assert that master is not included in the output of the validation.

Reply:
I will try that. Thanks!

Reply:
Fixed, I am just checking controller is present in both connections.